packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -157,6 +147,17 @@ function DashboardSidebarSubNavigation({
     [],
   );
 
+  const handleLinkClick = React.useMemo(() => {
```

Comment:
Why not a `.useCallback` here? Not saying it's wrong, just curious as it's what I would use.

Reply:
Because `useCallback` always returns a function, and here I want it to be able to be `undefined`